docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -364,3 +364,11 @@ function hs_paragraph_types_update_10009() {
     }
   }
 }
+
+/**
+ * Removes legacy background color field from collection and priv collection.
+ */
+function hs_paragraph_types_update_10010() {
+  FieldConfig::loadByName('paragraph', 'hs_collection', 'field_paragraph_style')->delete();
+  FieldConfig::loadByName('paragraph', 'hs_priv_collection', 'field_paragraph_style')->delete();
```

Comment:
```suggestion
if ($hs_collection_style = FieldConfig::loadByName('paragraph', 'hs_collection', 'field_paragraph_style')) {
  $hs_collection_style->delete();
}
if ($hs_priv_collection_style = FieldConfig::loadByName('paragraph', 'hs_priv_collection', 'field_paragraph_style')) {
  $hs_priv_collection_style->delete();
}
```

It's pretty safe to assume these exist, but doesn't hurt to check first just in case.